creator-node/src/routes/nodeSync.js

```diff
@@ -67,6 +68,25 @@ module.exports = function (app) {
       const ipfs = req.app.get('ipfsAPI')
       let ipfsIDObj = await getIPFSPeerId(ipfs, config)
 
+      // Ensure all relevant files are available through IPFS at export time
+      await Promise.all(files.map(async (file) => {
+        if (file.type === 'track' || file.type === 'metadata' || file.type === 'copy320') {
+          await rehydrateIpfsFromFsIfNecessary(
+            req,
+            file.multihash,
+            file.storagePath)
+        } else if (file.type === 'image') {
+          let sourcePath = file.sourceFile
+          if (sourcePath.includes('blob')) {
```

Comment:
do you only need to special case "blob" and not "Artwork"? not sure if other cases but these are the only ones i saw

Reply:
didn't see any artwork in files table or when i was uploading, can discuss offline

Reply:
update to this - i found a bunch of Artwork in the prod cnode files table soooo i think this _is_ required